src/app/lib/pageStyles/index.jsx

```diff
@@ -0,0 +1,15 @@
+import styled, { css } from 'styled-components';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+
+export const PageMarginStyles = css`
+  margin: 0 ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    margin: 0 ${GEL_SPACING_DBL};
+  }
+`;
+
+export const MainContainer = styled.main.attrs({ role: 'main' })`
```

Comment:
I suggest naming this way since Most Read, Front pages and IDX's are all index pages.

```suggestion
export const IndexMain = styled.main.attrs({ role: 'main' })`
```

Reply:
@sareh I've renamed it to IndexMain, but please note that this will only be used in the Most Read page. On the Front Page and IDX page, we add the margins to the PageContainer not to the main, since we want some sections to have a background that takes the full width of the page like for example the Ads or RadioSchedules.